### PR TITLE
add validations for warning and critical

### PR DESCRIPTION
**Pull request by vdepatla**
added validations for warning and critical read from environment

### CODE REVIEW

File: cmd/oomhero/main.go [hunk 1]
```diff
@@ -17,8 +17,24 @@ var (
 
 // reads warning and critical from environment or use the default ones.
 func init() {
-	warning = envVarToUint64("WARNING", warning)
-	critical = envVarToUint64("CRITICAL", critical)
+	warningEnv := envVarToUint64("WARNING", warning)
+	criticalEnv := envVarToUint64("CRITICAL", critical)
+
+	// validate warning and critical https://github.com/ricardomaraschini/oomhero/issues/4
```
Comment (ricardomaraschini) [inline cmd/oomhero/main.go]: I think it is safe to remove this comment.

File: cmd/oomhero/main.go [hunk 2]
```diff
@@ -17,8 +17,24 @@ var (
...
+	if warningEnv > 100 || criticalEnv > 100 {
+		log.Printf(
+			"warning and critical must be lower or equal to 100 .. using default values warning: %d critical: %d",
```
Comment (ricardomaraschini) [inline cmd/oomhero/main.go]: As we are already logging the values being used on `main()` I think we could omit the second part of this expression. You could leave it as:

`log.Print("warning and critical must be lower or equal to 100")`

What do you think?

File: cmd/oomhero/main.go [hunk 3]
```diff
@@ -17,8 +17,24 @@ var (
...
+			warning, critical)
+		return
+	} else if warningEnv > criticalEnv {
+		log.Printf(
+			"warning must be lower or equal to critical .. using default values warning: %d critical: %d",
```
Comment (ricardomaraschini) [inline cmd/oomhero/main.go]: Same as my previous comment.

File: cmd/oomhero/main.go (hunk 1 above)
Comment (vdepatla) [reply cmd/oomhero/main.go]: removed

File: cmd/oomhero/main.go (hunk 2 above)
Comment (vdepatla) [reply cmd/oomhero/main.go]: makes sense .. updated

**Comment by vdepatla** [comment]
@ricardomaraschini  updated .. pls take a look ..

**Pull request merged**